<?xml version="1.0"?>
<launch>

		<!-- Our node to control the drone -->
    <node pkg="px4_ground" type="control_vel.py" name="offb_node_py" required="true" output="screen" />

		<!-- Our control keyboard -->
    <node pkg="px4_ground" type="teleop_twist_keyboard.py" name="keyb_node_py" required="true" output="screen" />

		<!-- Wind plugin -->
    <!-- <node pkg="px4_ground" type="wind.py" name="offb_node_py" required="true" output="screen" /> -->

</launch>
<!-- Include the MAVROS node with SITL and Gazebo -->